pack.yaml

```diff
@@ -6,7 +6,11 @@ keywords:
   - issues
   - ticket management
   - project management
```

Comment:
@dylan-bitovi still looks like there is an issue here, looks like the merge boundaries leaked into your commit

Reply:
Fixed.
-D